rust_snuba/Cargo.toml

```diff
@@ -58,9 +58,3 @@ tikv-jemallocator = "0.5"
 [[bench]]
 name = "processors"
 harness = false
-
-[[bin]]
-# this is not a proper benchmark since it runs forever.
-name = "python-processor-infinite"
```

Comment:
can we move that benchmark instead of deleting it?

Reply:
sure, replaced it